audio/portaudio/files/patch-audacity-portmixer.diff

```diff
@@ -222,15 +230,15 @@ Last-Update: 2011-12-07
 --- src/hostapi/coreaudio/pa_mac_core_blocking.c
 +++ src/hostapi/coreaudio/pa_mac_core_blocking.c
 @@ -66,6 +66,9 @@
- #ifdef MOSX_USE_NON_ATOMIC_FLAG_BITS
- # define OSAtomicOr32( a, b ) ( (*(b)) |= (a) )
- # define OSAtomicAnd32( a, b ) ( (*(b)) &= (a) )
-+#elif MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_3
-+# define OSAtomicOr32( a, b ) BitOrAtomic( a, (UInt32 *) b )
-+# define OSAtomicAnd32( a, b ) BitAndAtomic( a, (UInt32 *) b )
- #else
- # include <libkern/OSAtomic.h>
- #endif
+ #ifdef MOSX_USE_NON_ATOMIC_FLAG_BITS
+ # define OSAtomicOr32( a, b ) ( (*(b)) |= (a) )
+ # define OSAtomicAnd32( a, b ) ( (*(b)) &= (a) )
++#elif MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_3
++# define OSAtomicOr32( a, b ) BitOrAtomic( a, (UInt32 *) b )
++# define OSAtomicAnd32( a, b ) BitAndAtomic( a, (UInt32 *) b )
+ #else
```

Comment:
I'm having trouble understanding what changed in this patch?

Reply:
To be perfectly honest, I couldn't say right now whether the change is strictly necessary because I actually made this a while ago (now I just fixed some minor things and made sure to lint it and such) but the source file changed, my guess is maybe the original patch failed with the new source?

Reply:
I can't even see what the change was. It looks like you just modified whitespace? I can't even tell.

Reply:
```diff
@@ -84,11 +84,26 @@
     */
     PaWinWaveFormatChannelMask channelMask;
 
-}PaWinDirectSoundStreamInfo;
+ }PaWinDirectSoundStreamInfo;
```
vs

```diff
@@ -86,6 +86,21 @@
 
-}PaWinDirectSoundStreamInfo;
+ }PaWinDirectSoundStreamInfo;
```

Reply:
I don't see what those lines you just posted have to do with the section I flagged. It seems like you're doing some no-ops with

`#elif MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_3`

And also, it isn't even clear to me that we care about 10.3 any more.

Reply:
Do you want me to remove the 10.3 thing and the patches targeting other OS? Both things are most likely superfluous but I'd be inclined to not fix what ain't really broken.

Reply:
I checked and it's merely about LF vs. CRLF line endings. The line endings were changed in upstream sources, so this patch has zero influence, it just needs to be there.